pkg/clientset_test/clientset_test.go

```diff
@@ -0,0 +1,29 @@
+package clientset_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	
```

Comment:
Not sure why but linter is complaining about this line 🤔 

https://github.com/Kong/kubernetes-ingress-controller/actions/runs/4297670409/jobs/7494491099#step:7:40

Do you mind checking this locally via `make lint` and `make verify.generators` to ensure we're good? Sorry for this to be a pain.

Reply:
There was some whitespace there. I've removed it and waiting for the build status now